app/scripts/react-directives/users/usersPage.jsx

```
@@ -0,0 +1,71 @@
+            disabled={store.pageWrapperStore.loading}
+            onClick={() => store.addUser()}
+          />
+        </div>
+      </PageTitle>
+      <PageBody loading={store.pageWrapperStore.loading}>
+        <table className="table">
+          <thead>
+            <tr>
+              <th>{t('users.labels.login')}</th>
+              <th>{t('users.labels.type')}</th>
+              <th></th>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)